build_docker_images.sh

```diff
@@ -0,0 +1,21 @@
+ingestion_image_name="ingestion-service"
```

Comment:
Ah, I actually already have a PR out to add basically this exact script here: https://github.com/SatcherInstitute/health-equity-tracker/pull/357

Do you want to just review that one and remove this file from this PR?

Reply:
Removed